x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/crawler/crawl_requests_panel/crawl_requests_panel.tsx

```diff
@@ -9,16 +9,47 @@ import React from 'react';
 
 import { useValues } from 'kea';
 
-import { EuiCode, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import { EuiButton, EuiCode, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import type { DiscoverStart } from '@kbn/discover-plugin/public';
 import { i18n } from '@kbn/i18n';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
 
 import { DataPanel } from '../../../../../shared/data_panel/data_panel';
 import { CrawlerLogic } from '../crawler_logic';
 
 import { CrawlRequestsTable } from './crawl_requests_table';
 
+interface KibanaDeps {
+  discover: DiscoverStart;
+}
+
+const CRAWLER_LOGS_DISCOVER_RECORD = {
+  dataViewSpec: {
+    name: i18n.translate(
+      'xpack.enterpriseSearch.crawler.crawlRequestsPanel.discoverCrawlerLogsTitle',
+      {
+        defaultMessage: 'All Crawler Logs',
+      }
+    ),
+    title: 'logs-elastic_crawler-default',
+  },
+  columns: [
+    '@timestamp',
+    'crawler.crawl.id',
+    'url.domain',
+    'url.path',
+    'event.action',
+    'http.response.status_code',
+  ],
+  sort: [['@timestamp', 'desc']],
+};
```

Comment:
do we want to put this somewhere else? it is a bit close to "magic value" territory but i'm not sure what/if there is a better way to handle it

Reply:
I think it's fine to leave it here for now until we find a need to re-use this in more places. Worth discussing a longer-term approach to linking to discover dashboards at some point.